pkg/domain/entities/images.go

```diff
@@ -183,8 +183,8 @@ type ImagePushOptions struct {
 	// SignBy adds a signature at the destination using the specified key.
 	// Ignored for remote calls.
 	SignBy string
-	// TLSVerify to enable/disable HTTPS and certificate verification.
-	TLSVerify types.OptionalBool
+	// SkipTLSVerify to skip HTTPS and certificate verification.
+	SkipTLSVerify types.OptionalBool
```

Comment:
Don't we have a `--tlsverify` and not a `--skiptlsverify` option?  This var name now seems to negate that to me.  Maybe it's my turn to be confused?

Reply:
The CLI option is parsed and set to another variable (not this one). c/image is using the skip semantics. We have this logic inverted with other commands as well (all leading to failing --tls-verify).

I opted for making the Skip logic consistent in the code. Will hopefully prevent us from doing too much brain yoga.